src/logic/wallets/ethTransactions.ts

```diff
@@ -41,11 +42,21 @@ export const calculateGasPrice = async () => {
     return '20000000000'
   }
 
-  const url = 'https://ethgasstation.info/json/ethgasAPI.json'
-  // const errMsg = 'Error querying gas station'
-  const { data } = await axios.get(url)
+  const gasPrice = getGasPrice()
+  const url = getGasPriceOracleUrl()
 
-  return new BigNumber(data.average).multipliedBy(1e8).toString()
+  if (gasPrice) {
+    // Fixed gas price in configuration. xDai uses this approach
+    return new BigNumber(gasPrice).toString()
+  } else if (url) {
+    // Fetch from gas price provider
+    const { data } = await axios.get(url)
+
+    return new BigNumber(data.average).multipliedBy(1e8).toString()
```

Comment:
mmm, how can you be sure the average will always be returned from the endpoint?
I would suggest two alternatives:
a. creating a function as we did for the explorer URL.
b. Add in the config another attribute that gives us the path to obtain the data from the endpoint result. For example `average` from the current endpoint or `standard` for this one https://station.energyweb.org/

Reply:
I added a solution using alternative b